proposals/0016-c-interface-compiler-library.md

```diff
@@ -148,9 +159,16 @@ struct IDxcResult : public IDxcOperationResult {
   DXC_OUT_KIND PrimaryOutput();
 };
 ```
+
 #### Helper library
 The shader compiler library also provides a helper library IDxcUtils that is
 used to create/consume the different data in the forms referred to above.
```

Comment:
```suggestion
#### IDxcUtils
The shader compiler library also provides a set of utility functions via IDxcUtils. This can be
used to create/consume the different data in the forms referred to above.
```

I really don't like calling this a "helper library" since it isn't a library in the .lib or .dll sense of the word.